### ISSUE TITLE
Alerts are too verbose when problems panel is open.

### ISSUE THREAD

**Issue by pawelurbanski** [microsoft/vscode#91166]
- VSCode Version: 1.42.1 - 1.43.0
- OS Version: Windows 10 Version 1909 (OS Build 18363.657)

Steps to Reproduce:

1. Open a file and toggle problems panel
2. Start coding and introduce some error
3. Remove all the errors from teh current file or workspace

* The screen reader will be reading every possible change reflected in the panel,
* Namely announcing the number of problems or lack of them,
* Those alerts are read after every keystroke which changes the number of problems shown

Does this issue occur when all extensions are disabled?: Yes/No
Yes it does - takes place regardless.

A solution to the problem:

The panel view message box, which displays constantly changing values has the aria-live attribute. It makes the screen reader read out every change.
This information is useless for only after focusing to the panel treeview one can do anything about them.
Remove this attribute located in this place:
https://github.com/microsoft/vscode/blob/8451f8bd913914aac17b6b3273b6ed0db137a217/src/vs/workbench/contrib/markers/browser/markersView.ts#L352

There will not be this spammy behavior. All the values for the number of problems found will still be available when toggling focus between the editor and the problems panel view.

@isidorn  I skipped creating a full PR, since it is just a single line change anyone on the team can do. Should take less time than merging a PR.

**Comment by isidorn** [comment]
This makes snese to me. And we can remove the `aria-live` attribute.
@sandy081 let me know if you do not mind and I will just remove that code line?

**Comment by sandy081** [comment]
👍 Please go ahead.
